NEWS.md

```diff
@@ -7,6 +7,10 @@ Note that each entry is kept to a minimum, see links for details.
 
 ## Language changes
 
+* A new `performance` warning category was introduced.
+  They are not displayed by default even in verbose mode.
+  Turn them on with `-wW:performance` or `Warning[:performance] = true`. [Feature #19538]
```

Comment:
```suggestion
  Turn them on with `-W:performance` or `Warning[:performance] = true`. [Feature #19538]
```

Reply:
I wonder, did you maybe mean teh `-wW:performance` here? Are performance warnings emitted with VERBOSE=false or they VERBOSE=true (in addition to the category being enabled). It seems best to emit them at VERBOSE=false level, so it's much easier to enable them, since they are already opt-in.

Reply:
Maybe I misinterpreted https://bugs.ruby-lang.org/issues/19538#note-9 ?

>  said that ruby -w should not enable this warning mode. ruby -W:performance should do.